### ISSUE TITLE
Telegram c++ library

### ISSUE THREAD

**Issue by lukap3rcic** [telegramdesktop/tdesktop#2006]
Since [Telegram CLI for Linux](https://github.com/vysheng/tg) is dead, can you help us with releasing Telegram c++ library (or at least MTProto library)?

Thnx,
-L

**Comment by stek29** [comment]
I think Preston will answer your questions if he has time and you actually ask.
So please, ask what you want. Don't ask "Can I ask".

**Comment by lukap3rcic** [comment]
Fair point- Let me rephrase that;
Will you release Telegram c++ library (or at least MTProto library)?

**Comment by stek29** [comment]
I don't think Developer has time for it. Maybe you can do it?

**Comment by lukap3rcic** [comment]
@stek29 
I found this gem;
http://aseman.co/en/products/developer/telegram/

They use [launchapd for issues (and not github)](https://launchpad.net/telegram-app).

It looks very clean (only dependencies are qt and OpenSSL).

I think this repo should be featured on [Telegram site under #unofficial-apps](https://telegram.org/apps#unofficial-apps) instead of now dead Telegram cli (c++).
